epic.go

```diff
@@ -116,15 +149,10 @@ type leaderboardEntry struct {
 
 // QueryPlayer looks up a player by their username and platform, and returns information about that player, namely, the
 // statistics for the 3 different party modes.
-func (s *Session) QueryPlayer(name string, accountId string, platform string) (*Player, error) {
+func (s *Session) QueryPlayer(name string, accountId string) (*Player, error) {
```

Comment:
I'm not the biggest fan of empty arguments. Perhaps it should be `QueryPlayerById()` and `QueryPlayerByName()`? What do you think?